core/core.go

```diff
@@ -382,31 +382,76 @@ func makeAddrsFactory(cfg config.Addresses) (p2pbhost.AddrsFactory, error) {
 	}, nil
 }
 
-func makeSmuxTransport(mplexExp bool) smux.Transport {
+func makeSmuxTransport(multiplexers []string, mplexExp bool) smux.Transport {
 	mstpt := mssmux.NewBlankTransport()
 
-	ymxtpt := &yamux.Transport{
-		AcceptBacklog:          512,
-		ConnectionWriteTimeout: time.Second * 10,
-		KeepAliveInterval:      time.Second * 30,
-		EnableKeepAlive:        true,
-		MaxStreamWindowSize:    uint32(1024 * 512),
-		LogOutput:              ioutil.Discard,
+	multiplexerProtocolNames := map[string]string{
+		"yamex": "/yamux/1.0.0",
```

Comment:
Any reason for it being called `yamex` instead of `yamux`?

Reply:
oh wow thanks for catching that.  im not sure why i did that (i also spelled it with an a above 😬).  ill fix this right now.